src/navigation/navigation.es6

```diff
@@ -83,6 +83,7 @@ const destroy_windows = (data_attribute) => {
    state.switchAccount = (id) => {
       destroy_windows('data-account-specific=true');
       liveapi.switch_account(id)
+         .then(() => window.location.reload())
```

Comment:
it was used for testing because i realized that the chart draw session is messed up after switching the account, reloading fix this problem. ill try now without reload and see what happens